config/conference.ts

```diff
@@ -151,18 +151,15 @@ const Conference: IConference = {
 
   Socials: {
     Twitter: {
-      Id: '977876011',
-      Name: 'DDDPerth',
+      Id: '?',
```

Comment:
According to https://www.codeofaninja.com/tools/find-twitter-id/, the id for DDDMelb is `971044039`. Is this for the twitter API? Do we need to worry about billing in the longer term there?

Reply:
Thanks Gaving, updated it! It's only used for composing a message from `twitter.com`, so should be subject to any charges, hopefully.